plugin/build.gradle.kts

```
@@ -32,10 +32,15 @@ dependencies {
      */
 
     testImplementation(gradleTestKit())
-    testImplementation("junit:junit:${PluginVersions.junit}")
+    testImplementation("org.junit.jupiter:junit-jupiter:${PluginVersions.junit5}")
+    testImplementation("org.assertj:assertj-core:${PluginVersions.assertJ}")
     testImplementation(kotlin("reflect"))
 }
 
+tasks.withType<Test> {
```

This magical incantation is preferred over that one you have there.

```suggestion
tasks.withType<Test>().configureEach {
```

The one you have will eagerly configure the `Test` task. The one in the suggestion will be only be executed when needed.